### PR TITLE
Ddpb 1772

### PR DESCRIPTION
**Pull request by elvisciotti**

### CODE REVIEW

File: src/AppBundle/Controller/Admin/AdController.php [hunk 1]
```diff
@@ -37,7 +37,6 @@ public function indexAction(Request $request)
                                  , new EntityDir\User()
                                  , [ 'options' => [ 'roleChoices'        => [EntityDir\User::ROLE_LAY_DEPUTY=>'Lay deputy'],
```
Comment (jackgoodby) [inline src/AppBundle/Controller/Admin/AdController.php]: or just make the third param a simple array: 

 ['roleChoices'        => [EntityDir\User::ROLE_LAY_DEPUTY=>'Lay deputy'],
'roleNameSetTo'      => EntityDir\User::ROLE_LAY_DEPUTY,
'roleNameEmptyValue' => null,]

File: src/AppBundle/Form/Ad/AddUserType.php [hunk 2]
```diff
@@ -8,54 +8,30 @@
...
-
         $builder
             ->add('firstname', 'text')
             ->add('lastname', 'text')
-            ->add('roleName', 'choice', $roleNameOptions)
-            ->add('odrEnabled', 'checkbox', $odrEnabledOptions)
+            ->add('roleName', 'choice', [
+                    'empty_value' => null,
+                    'choices'     => $options['options']['roleChoices'],
+                    'data'        => $options['options']['roleNameSetTo'],
```
Comment (jackgoodby) [inline src/AppBundle/Form/Ad/AddUserType.php]: If you're passing a single array of options these could be $options['roleChoices'], $options['roleNameSetTo'] etc

Comment (elvisciotti) [reply src/AppBundle/Form/Ad/AddUserType.php]: yes, exactly what I'm planning to simplify after I write a behat test (that will guarantee I don't mess it up)

File: src/AppBundle/Controller/Admin/AdController.php (hunk 1 above)
Comment (elvisciotti) [reply src/AppBundle/Controller/Admin/AdController.php]: I'll clean it up further after I write the test, I'll submit for a 2nd review later

**Pull request merged**